man/systemd-nspawn.xml

```diff
@@ -884,6 +884,25 @@
         <literal>auto</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--timezone=</option></term>
+
+        <listitem><para>Configures how <filename>/etc/localtime</filename> inside of the container (i.e. local timezone
+        synchronization from host to container) shall be handled. Takes one of <literal>off</literal>,
+        <literal>copy</literal>, <literal>bind</literal>, <literal>symlink</literal>, <literal>delete</literal> or
+        <literal>auto</literal>. If set to <literal>off</literal> the <filename>/etc/localtime</filename> file in the
+        container is left as it is included in the image, and neither modified nor bind mounted over. If set to
```

Comment:
"is left as it is included in the image, and neither modified nor bind mounted over" → "is left untouched". Shorter but easier to read, and then later on overwriting and bind-mounting is talked about anyway, so no need to repeat this here.